tests/_data/schema.sql

```diff
@@ -73,6 +73,7 @@ CREATE TABLE `user` (
   `auth_tf_enabled` tinyint(1) DEFAULT '0',
   `flags` int(11) NOT NULL DEFAULT '0',
   `last_login_at` int(11) DEFAULT NULL,
+  `last_password_change` timestamp,
```

Comment:
can we updated to the name `password_changed_at` and use the type `int` as the rest of the time fields on the table?

Reply:
done!